test/support/test_config.h

```diff
@@ -116,4 +102,17 @@
 
 #define _PSTL_MSVC_LESS_THAN_CPP20_COMPLEX_CONSTEXPR_BROKEN (_MSC_VER && __cplusplus <= 201703L)
 
+#define _PSTL_ICC_TEST_COMPLEX_ASIN_MINUS_INF_NAN_BROKEN_SIGNBIT __INTEL_LLVM_COMPILER
+#define _PSTL_ICC_TEST_COMPLEX_COSH_MINUS_INF_MINUS_ZERO_BROKEN_SIGNBIT __INTEL_LLVM_COMPILER
+#define _PSTL_ICC_TEST_COMPLEX_COSH_MINUS_ZERO_MINUS_ZERO_BROKEN_SIGNBIT __INTEL_LLVM_COMPILER
+#define _PSTL_ICC_TEST_COMPLEX_POW_COMPLEX_COMPLEX_PASS_BROKEN_TEST_EDGES __INTEL_LLVM_COMPILER
+#define _PSTL_ICC_TEST_COMPLEX_POW_COMPLEX_SCALAR_PASS_BROKEN_TEST_EDGES __INTEL_LLVM_COMPILER
+#define _PSTL_ICC_TEST_COMPLEX_POW_SCALAR_COMPLEX_PASS_BROKEN_TEST_EDGES __INTEL_LLVM_COMPILER
+#define _PSTL_ICC_TEST_COMPLEX_NORM_MINUS_INF_NAN_BROKEN_TEST_EDGES __INTEL_LLVM_COMPILER
+#define _PSTL_ICC_TEST_COMPLEX_POLAR_BROKEN_TEST_EDGES __INTEL_LLVM_COMPILER
+
+#define _PSTL_CLANG_TEST_COMPLEX_ACOS_IS_NAN_CASE_BROKEN __clang__
+#define _PSTL_CLANG_TEST_COMPLEX_ATAN_IS_CASE_BROKEN __clang__
+#define _PSTL_CLANG_TEST_COMPLEX_SIN_IS_CASE_BROKEN __clang__
+

```

Comment:
Claiming that all versions of a certain compiler are "broken" is not quite good in my opinion. It is better to set an upper bound on the version, e.g. "< the next not yet released version"

Also, do all these macros indicate different compiler problems, or supposedly the same problem? If the same problem, there should only be one macro.

Last but not least, were bugs submitted for these issues?

Reply:
"Also, do all these macros indicate different compiler problems, or supposedly the same problem? If the same problem, there should only be one macro." - fixed in https://github.com/oneapi-src/oneDPL/pull/734